src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,96 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
+import { getUser } from '../../api';
+
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+interface Props {
+  isSelectedTodo: Todo,
+  setIsSelectedTodoId: (todoId: number) => void;
+  isLoadedUser: boolean;
+  setIsLoadedUser: (status: boolean) => void;
+}
+
+export const TodoModal: React.FC<Props> = ({
+  isSelectedTodo,
+  setIsSelectedTodoId,
+  isLoadedUser,
+  setIsLoadedUser,
+}) => {
+  const [user, setUser] = useState<User | null>(null);
+  const {
+    id,
+    title,
+    completed,
+    userId,
+  } = isSelectedTodo;
+
+  useEffect(() => {
+    const getUserFromServer = async () => {
+      try {
+        const userFromServer = await getUser(userId);
+
+        setUser(userFromServer);
+        setIsLoadedUser(true);
+      } catch (error) {
+        throw new Error('Failed on loading user from server');
+      }
```

Comment:
What's the point in throwing error if error is thrown before? You need to show user that this info is not loaded and there is some error, user will not open console (in most cases user don't know what is console)

Let's show it on UI instead, you have modal with todo info, so you can show red text with some error text there